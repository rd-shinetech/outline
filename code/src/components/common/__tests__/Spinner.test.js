import React from 'react';
import {shallow} from 'enzyme';
import toJson from 'enzyme-to-json';
import Spinner from '../Spinner';



describe('Spinner.test.js', () => {
    let wrapper = undefined;

    beforeEach(() => {
        wrapper = shallow(<Spinner/>);
        return wrapper;
    });


    it('renders without crashing', () => {
        expect(wrapper.length).toEqual(1);
    });


    it('renders as expected', () => {
        const tree = toJson(wrapper);
        expect(tree).toMatchSnapshot();
    });    
});
