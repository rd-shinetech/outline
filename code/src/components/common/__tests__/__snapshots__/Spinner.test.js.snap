exports[`Spinner.test.js renders as expected 1`] = `
<h3
  className="bg-info text-white font-weight-bold">
  .
  Â 
</h3>
`;
